compiler/qsc_codegen/src/qsharp/spec_decls.rs

```diff
@@ -36,14 +36,14 @@ fn attributes() {
         indoc! {r#"
             namespace Sample {
                 @EntryPoint()
-                @Config(Base)
+                @Config(Unrestricted)
```

Comment:
Why did this test have to change?

Reply:
Because a bug was fixed in contig attr logic.